packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -88,7 +88,25 @@ export async function storeEvents(
   config: BoosterConfig,
   logger: Logger
 ): Promise<void> {
-  // TODO: Manage query pagination and db.batchWrite limit of 25 operations at a time
+  const batchItemsNumbers = 25
+  let processedItems = 0
+  while (Math.floor((eventEnvelopes.length - processedItems) / batchItemsNumbers) != 0) {
+    const currentBatch = eventEnvelopes.slice(processedItems, processedItems + batchItemsNumbers)
+    await persistBatch(logger, currentBatch, config, dynamoDB)
+    processedItems += batchItemsNumbers
+  }
```

Comment:
Can you extract the `while` condition into a separate function, and the body as well? It's a bit difficult to read what's going on here

Reply:
I ended refatoring the wway it was done :P